pkg/spdk/replica.go

```diff
@@ -209,6 +209,12 @@ func NewReplica(ctx context.Context, replicaName, lvsName, lvsUUID string, specS
 	}
 }
 
+func (r *Replica) GetAddress() string {
+	r.RLock()
+	defer r.RUnlock()
+	return fmt.Sprintf("tcp://%s:%d", r.IP, r.PortStart)
```

Comment:
NIT: To avoid confusion or misleading, I prefer to let the caller to add prefix `tcp://` , or change the function name

Reply:
Updated. Thank you.